.gitignore

```diff
@@ -24,7 +24,7 @@ tensorboard/*
 onnx/*
 repos/*
 
-/docs/source/api/*.rst
+/docs/api/*.rst
```

Comment:
Need to remove, we shouldn't be ignoring anything for the api docs rst files now